app/uk/gov/hmrc/helptosavefrontend/util/Toggles.scala

```diff
@@ -16,58 +16,50 @@
 
 package uk.gov.hmrc.helptosavefrontend.util
 
-import java.time.Instant
-
+import com.typesafe.config.Config
 import play.api.{Configuration, Logger}
 
 object Toggles {
 
-  case class FEATURE[A](name: String, conf: Configuration, unconfiguredVal: Option[A], logger: Logger) {
```

Comment:
this file was activating the `Throw` wart and when it was being used it was activating the `NonUnitStatements` wart. Since these two warts were inherent in the design I've replaced it with a cleaner implementation with no warts